src/IECoreImage/DisplayDriverServer.cpp

```diff
@@ -173,6 +173,13 @@ class DisplayDriverServer::PrivateData : public RefCounted
 			m_endpoint = boost::asio::ip::tcp::endpoint( tcp::v4(), portNumber );
 			m_acceptor.open(  m_endpoint.protocol() );
 			m_acceptor.set_option( boost::asio::ip::tcp::acceptor::reuse_address( true ) );
```

Comment:
I think we want to delete this line, as `reuse_address` is now present in both the conditional blocks below?

Reply:
Definitely, fixed in ced591d8b6e3bf9c715775561bf1f9eaed355a15